internal/controller/postgrescluster/postgres.go

```diff
@@ -628,6 +635,71 @@ func (r *Reconciler) reconcilePostgresDataVolume(
 	return pvc, err
 }
 
+// determineVolumeSize compares the potential sizes from the instance spec, status
+// and limit and sets the appropriate current value.
```

Comment:
:pick: This method seems more accurately described as `setVolumeSize`.

Reply:
That works for me, I'll make that change.